using/solving-exercises/tdd.md

```diff
@@ -0,0 +1,85 @@
+# Test-Driven Development (TDD)
+
+Test-Driven Development (sometimes called Test-First Development or Test-Driven Design) is the practice of writing the unit tests first, before you write a single line of implementation code.
+
+## On Exercism, the tests _are_ the requirements!
+
+All Practice Exercises you work on (those ones that don't teach you a new concept) will have some instructions describing in general terms what you need to do.
+By design, these instructions do not account for programming-language-specific implementation details because they are shared by all of Exercism’s 70+ language tracks.
```

Comment:
This line idea a smart apostrophe. Line 25 uses a single quote. It would be good to be consistent with the contractions.

Reply:
Line 7 as well, so even "close" example of inconsistency.